docs/docs/site-docs.js

```diff
@@ -23034,72 +23026,6 @@
 	    };
 	
-	    __exportStar = function (m, exports) {
-	        for (var p in m) if (!exports.hasOwnProperty(p)) exports[p] = m[p];
-	    };
-	
-	    __values = function (o) {
-	        var m = typeof Symbol === "function" && o[Symbol.iterator], i = 0;
-	        if (m) return m.call(o);
-	        return {
-	            next: function () {
-	                if (o && i >= o.length) o = void 0;
-	                return { value: o && o[i++], done: !o };
-	            }
-	        };
-	    };
-	
-	    __read = function (o, n) {
-	        var m = typeof Symbol === "function" && o[Symbol.iterator];
-	        if (!m) return o;
-	        var i = m.call(o), r, ar = [], e;
-	        try {
-	            while ((n === void 0 || n-- > 0) && !(r = i.next()).done) ar.push(r.value);
-	        }
-	        catch (error) { e = { error: error }; }
-	        finally {
-	            try {
-	                if (r && !r.done && (m = i["return"])) m.call(i);
-	            }
-	            finally { if (e) throw e.error; }
-	        }
-	        return ar;
-	    };
-	
-	    __spread = function () {
-	        for (var ar = [], i = 0; i < arguments.length; i++)
-	            ar = ar.concat(__read(arguments[i]));
-	        return ar;
-	    };
-	
-	    __await = function (v) {
-	        return this instanceof __await ? (this.v = v, this) : new __await(v);
-	    };
-	
-	    __asyncGenerator = function (thisArg, _arguments, generator) {
-	        if (!Symbol.asyncIterator) throw new TypeError("Symbol.asyncIterator is not defined.");
-	        var g = generator.apply(thisArg, _arguments || []), i, q = [];
-	        return i = {}, verb("next"), verb("throw"), verb("return"), i[Symbol.asyncIterator] = function () { return this; }, i;
-	        function verb(n) { if (g[n]) i[n] = function (v) { return new Promise(function (a, b) { q.push([n, v, a, b]) > 1 || resume(n, v); }); }; }
-	        function resume(n, v) { try { step(g[n](v)); } catch (e) { settle(q[0][3], e); } }
-	        function step(r) { r.value instanceof __await ? Promise.resolve(r.value.v).then(fulfill, reject) : settle(q[0][2], r);  }
-	        function fulfill(value) { resume("next", value); }
-	        function reject(value) { resume("throw", value); }
-	        function settle(f, v) { if (f(v), q.shift(), q.length) resume(q[0][0], q[0][1]); }
-	    };
-	
-	    __asyncDelegator = function (o) {
-	        var i, p;
-	        return i = {}, verb("next"), verb("throw", function (e) { throw e; }), verb("return"), i[Symbol.iterator] = function () { return this; }, i;
-	        function verb(n, f) { if (o[n]) i[n] = function (v) { return (p = !p) ? { value: __await(o[n](v)), done: n === "return" } : f ? f(v) : v; }; }
-	    };
-	
-	    __asyncValues = function (o) {
-	        if (!Symbol.asyncIterator) throw new TypeError("Symbol.asyncIterator is not defined.");
-	        var m = o[Symbol.asyncIterator];
-	        return m ? m.call(o) : typeof __values === "function" ? __values(o) : o[Symbol.iterator]();
-	    };

```

Comment:
wondering why all these lines are removed? @gscshoyru can you `yarn && yarn bootstrap` to make sure you have latest deps, then rebuild docs site and see if this diff persists?

Reply:
I did that, and nothing seems to have changed. I even ran the prepare release script to be sure. Possibly it's... something to do with my machine? I don't know why I'd be making such a difference.